src/relay/backend/te_compiler.cc

```diff
@@ -565,7 +578,8 @@ class LowerTensorExprMutator : public ExprMutator {
     }
 
     // Lower the primitive function for that target.
-    std::pair<GlobalVar, Attrs> pair = LowerFunction(prim_func, target);
+    Function func = GetRef<Function>(prim_func.as<FunctionNode>());
```

Comment:
Just Downcast<Function>(prim_func) will work here.

Reply:
Thanks, I haven't fully learnt the TVM standard library yet :smile_cat: